net-ldap.gemspec

```diff
@@ -29,6 +29,8 @@ the most recent LDAP RFCs (4510-4519, plutions of 4520-4532).}
   s.required_ruby_version = ">= 2.0.0"
   s.summary = %q{Net::LDAP for Ruby (also called net-ldap) implements client access for the Lightweight Directory Access Protocol (LDAP), an IETF standard protocol for accessing distributed directory services}
 
+  s.add_runtime_dependency("connection_pool", "~> 2.2")
```

Comment:
I'm not sure if we want to make this a requirement or allow users to opt in by adding the gem in their Gemfile. If we do that, then we have to handle when the library is missing.